eva_sub_cli/validators/docker_validator.py

```diff
@@ -38,21 +38,23 @@ def _validation_file_path_for(file_path):
 
     def get_docker_validation_cmd(self):
         if self.metadata_xlsx and not self.metadata_json:
-            docker_cmd = (
-                f"{self.docker_path} exec {self.container_name} nextflow run eva_sub_cli/nextflow/validation.nf "
-                f"--base_dir {container_validation_dir} "
-                f"--vcf_files_mapping {self.mapping_file} "
-                f"--metadata_xlsx {self.metadata_xlsx} "
+            docker_cmd = ''.join([
+                f"{self.docker_path} exec {self.container_name} nextflow run eva_sub_cli/nextflow/validation.nf ",
+                f"--base_dir {container_validation_dir} ",
+                f"--vcf_files_mapping {self.mapping_file} ",
+                f"--metadata_xlsx {self.metadata_xlsx} ",
+                f"--shallow_validation true " if self.shallow_validation else "",
```

Comment:
Feels like we could reduce some code repetition by doing this for the xlsx/json check as well, might be messier though - up to you.

Reply:
I changed it and tried to keep it readable